lib/Image/Image.js

```diff
@@ -134,26 +133,25 @@ module.exports = kind(
 		{from: '.position', to: '.$.image.position'}
 	],
 
-	/**
-	* Only generate `this.$.client` if the instance has components.
-	*
-	* @method
-	* @private
-	*/
 	create: function () {
-		Control.prototype.create.apply(this, arguments);
-		if (this.getClientControls().length > 0) {
-			this.$.client.canGenerate = true;
-			this.addClass('has-children');
+		// Adapting from the prior API. We'll assume that the presence of components but not
+		// overlayComponents means the consumer is expecting the former ImageBadge-style overlay
+		if (this.components && !this.overlayComponents) {
+			this.overlayComponents = this.components;
+			this.components = null;
+			this.overlayComponents.forEach(function (c) {
+				c.kind = c.kind || Icon
+			});
+			this.overlayPosition = this.overlayPosition || 'bottom';
```

Comment:
It can be changed by specifying `overlayPosition` which is the new API. This block is just for adapting from the old API so I do want to expose anything that perpetuates it.